### PR TITLE
Content Preview Window

### PR DESCRIPTION
**Pull request by edwardstamps**

### CODE REVIEW

File: web/admin/src/SearchBar.js [hunk 1]
```diff
@@ -16,8 +16,8 @@ export default class SearchBar extends Component {
 
   render() {
     return (
-      <div style={{margin: 5}}>
-        <input type="text" id="myInput" value={this.state.value} onChange={this.handleChange}/>
+      <div style={{margin: 5, marginLeft: 30, marginTop: 7}}>
```
Comment (AdamLiechty) [inline web/admin/src/SearchBar.js]: styles --> CSS

File: web/admin/src/ContentDetailsEditor.js [hunk 2]
```diff
@@ -53,7 +53,16 @@ export default class ContentDetailsEditor extends PureComponent {
...
       case 'survey': return <div>
-        TODO: Survey UI
+        <div>
+            <h2>Add a Title</h2>
+            <TextEditor content={c} prop="title" title="Title" placeholder="Acme Co Name" onUpdate={onUpdate} />
+          </div>
+          <div className="homeBox">
+            <h2>Add Content</h2>
+            <TextEditor content={c} prop="text" title="Content" placeholder="Acme Co Details" onUpdate={onUpdate} />
+            <TextEditor content={c} prop="url" title="URL" placeholder="http://www.acme.com" onUpdate={onUpdate} />
```
Comment (AdamLiechty) [inline web/admin/src/ContentDetailsEditor.js]: These looks like web view fields, not survey fields.

Comment (edwardstamps) [reply web/admin/src/ContentDetailsEditor.js]: I'm not familiar with our surveys but do they have titles and content already to match the mock ups?

Comment (AdamLiechty) [reply web/admin/src/ContentDetailsEditor.js]: Correct, all that we will need here is a selector to pick from the global surveys from the CMS

File: web/admin/src/CurrentContent.js [hunk 3]
```diff
@@ -1,24 +1,130 @@
...
+display: "flex",
+flexFlow: "row wrap",
+alignItems: "center",
+margin: '-1 -1 0 -1',
+height: 40,
+padding: 0,
+border: "1px solid #e2e2e2",
+textAlign: "center",
+borderRadius: 4,
+fontFamily: "-apple-system, BlinkMacSystemFont, 'Fira Sans', 'Open Sans', 'Helvetica Neue', sans-serif",
```
Comment (AdamLiechty) [inline web/admin/src/CurrentContent.js]: Why the font override?

File: web/admin/src/CurrentContent.js [hunk 4]
```diff
@@ -1,24 +1,130 @@
...
+
+const reorder = (list, startIndex, endIndex) => {
+  const result = Array.from(list);
+  const [removed] = result.splice(startIndex, 1);
+  result.splice(endIndex, 0, removed);
+  return result;
+};
+
+// using some little inline style helpers to make the app look okay
+const getItemStyle = (draggableStyle, isDragging) => ({
```
Comment (AdamLiechty) [inline web/admin/src/CurrentContent.js]: Let's have these styles in CSS, controlled by `className`s

File: web/admin/src/CurrentContent.js [hunk 5]
```diff
@@ -1,24 +1,130 @@
...
+      result.destination.index
+    );
+
+    this.setState({
+      items,
+    });
+   
+  }
+
+  // render() {
```
Comment (AdamLiechty) [inline web/admin/src/CurrentContent.js]: remove?

File: web/admin/src/CurrentContent.js [hunk 6]
```diff
@@ -1,24 +1,130 @@
...
-          <SearchBar updateList={this.props.updateList}/>
-        </span>
-        <ul className="current-content__list">
-          { content.map(c => <li key={c.key}>
-            <img src={iconFor(c)} className="current-content__icon" alt={c.type} />
-            <span className="current-content__title">{titleFor(c)}</span>
-            <Link to={`/content/${c.key}`} className="current-content__view">View</Link>
-          </li>)}
-        </ul>
+      <span className="content-bar">
```
Comment (AdamLiechty) [inline web/admin/src/CurrentContent.js]: looks like the tabs got messed up here?

**Pull request merged**